src/services/users.services.js

```diff
@@ -0,0 +1,41 @@
+const { User } = require('../models/User.model');
+
+const getUsers = async () => {
+  const users = await User.findAll({ order: ['name'] });
+
+  return users;
+};
+
+const getUserById = async (id) => {
+  const user = await User.findByPk(id);
+
+  return user;
+};
+
+const createUser = async (name) => {
+  const user = await User.create({ name: name });
+
+  return user;
+};
+
+const updateUser = async (id, name) => {
+  await User.update({ name }, { where: { id } });
+
+  return { id, name };
```

Comment:
The `updateUser` function returns an object with `id` and `name`, but it does not fetch the updated user from the database. Consider returning the updated user object from the database for consistency with other service functions and to ensure the returned data reflects the actual state in the database.